lib/SILGen/SILGenDistributed.cpp

```diff
@@ -209,7 +209,7 @@ void SILGenFunction::emitDistActorIdentityInit(ConstructorDecl *ctor,
       { temp, selfMetatypeValue });
 
   // --- initialize the property.
-  initializeProperty(*this, loc, borrowedSelfArg, var, temp);
```

Comment:
The issue was here, so changing to take here is correct, but the other sites want to remain not-take.

Thanks @eeckstein and @drexin for the pointers !